### PR TITLE
Add anyscript support (*.any)

### PR DESCRIPTION
**Pull request by melund**
**Changes proposed:**
* [x] Add

### CODE REVIEW

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export const extensions: IFileCollection = {
     { icon: 'ai', extensions: ['ai'], format: FileFormat.svg },
     { icon: 'ai2', extensions: ['ai'], format: FileFormat.svg, disabled: true },
     { icon: 'angular', extensions: ['.angular-cli.json', 'angular-cli.json'], filename: true, format: FileFormat.svg },
+    { icon: 'anyscript', extensions: ['any'], format: FileFormat.svg },
```
Comment (jens1o) [inline src/icon-manifest/supportedExtensions.ts]: add `languages: [languages.any]`

**Comment by JimiC** [comment]
Also, PR's from master branch` is not allowed.

File: src/icon-manifest/languages.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { ILanguageCollection } from '../models';
 export const languages: ILanguageCollection = {
   actionscript: { ids: ['nextgenas'], defaultExtension: 'as' },
   ansible: { ids: ['ansible'], defaultExtension: 'ansible' },
+  anyscript: { ids: ['anyscript'], defaultExtension: 'any'},
```
Comment (JimiC) [inline src/icon-manifest/languages.ts]: `anyscript` needs to added in `ILanguageCollection` too.

Comment (jens1o) [inline src/icon-manifest/languages.ts]: That's the class name. It's already added there?

Comment (JimiC) [inline src/icon-manifest/languages.ts]: @jens1o `ILanguageCollection` is the `interface` not the `class`.

Comment (jens1o) [inline src/icon-manifest/languages.ts]: Holy me...

File: src/icon-manifest/supportedExtensions.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export const extensions: IFileCollection = {
     { icon: 'ai', extensions: ['ai'], format: FileFormat.svg },
     { icon: 'ai2', extensions: ['ai'], format: FileFormat.svg, disabled: true },
     { icon: 'angular', extensions: ['.angular-cli.json', 'angular-cli.json'], filename: true, format: FileFormat.svg },
-    { icon: 'anyscript', extensions: ['any'], format: FileFormat.svg },
+    { icon: 'anyscript', languages: [languages.ansible], format: FileFormat.svg },
```
Comment (jens1o) [inline src/icon-manifest/supportedExtensions.ts]: `ansible`?

**Comment by melund** [comment]
Thanks for the feedback and your patience. I was a little unsure how the different files play together. I will close this and open a new PR which is not from master.

**Pull request closed without merging**